public/index.ts

```diff
@@ -1011,6 +1008,7 @@ function createTestVolume() {
 }
 
 async function createLoader(data: TestDataSpec): Promise<IVolumeLoader> {
+  await loadWorker.onOpen();
```

Comment:
because OpenCell is such a special case, maybe it should be checked first before even touching loadWorker?